app/src/main/kotlin/no/nav/arbeidsgiver/notifikasjon/infrastruktur/kafka/Config.kt

```diff
@@ -20,6 +22,8 @@ const val TOPIC = "fager.notifikasjon"
 
 private val strictObjectMapper = jacksonObjectMapper().apply {
     registerModule(JavaTimeModule())
+    disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
```

Comment:
ref: 3b08060a7f46b0767047131edd19f0b0f074108b

> usikker på om dette er et problem utenfor testene. er også usikker på om dette er beste fiks. 
endring av jackon config må testes i dev på eksisterende topic. tbd..

Reply:
replay fungerer som det skal. 
<img width="321" alt="image" src="https://user-images.githubusercontent.com/189395/164431633-64098c0c-6f9c-4904-990c-d9e31df6d8c0.png">

Reply:
Fra det jeg kan skjønne skal ikke denne endringen ødelegge noe.
Eksisterende datoer er skrevet ned som timestamps, og blir da lest opp med tz fra context. Nye datoer får da tz skrevet ned med iso format, og siden vi nå disabler `ADJUST_DATES_TO_CONTEXT_TIME_ZONE` så vil en evt. tidsone som er serialisert også bli videreført til det deserialiserte objektet.

> ADJUST_DATES_TO_CONTEXT_TIME_ZONE: Feature that specifies whether context provided java.util.TimeZone (DeserializationContext.getTimeZone() should be used to adjust Date/Time values on deserialization, even if value itself contains timezone information. If enabled, contextual TimeZone will essentially override any other TimeZone information; if disabled, it will only be used if value itself does not contain any TimeZone information.